helidon-ls-ext/README.md

```diff
@@ -0,0 +1,13 @@
+# Helidon.ls.ext
```

Comment:
Looking around, I see this https://github.com/eclipse/lsp4mp
Maybe this new module should be called `lsp4helidon`. Note that I'd like to remove the `helidon-` from the top level directories sometime soon, so it be nice to use a name that does not start with `helidon-` to avoid a future refactoring.

Reply:
done